packages/admin/cms-admin/src/dam/DataGrid/fileUpload/FileUploadContext.tsx

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 
+import { FileWithFolderPath } from "./useDamFileUpload";
+
 export interface NewlyUploadedItem {
     id: string;
     parentId?: string;
     type: "file" | "folder";
+    file?: FileWithFolderPath;
```

Comment:
Why do you need the file here?

Reply:
To match the uploaded files to the damFile ID directly in the admin. Otherwise I only have the file id and need to make a request to get the name.